src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -52,9 +71,14 @@ function AttachmentCarouselPager({items, renderItem, initialIndex, onPageSelecte
     }, []);
 
     useEffect(() => {
-        setActivePageState(initialIndex);
-        activePage.value = initialIndex;
-    }, [activePage, initialIndex]);
+        setActivePageIndex(initialPage);
+        activePage.value = initialPage;
+    }, [activePage, initialPage]);
+
+    /**
+     * The pager uses the source index and current active state to render the pages.
+     */
```

Comment:
```suggestion
```
This isn't a valuable comment IMO. The code is clear enough.

Reply:
As a general rule of thumb, code comments should explain _why_ the logic is what it is, and comments that only explain _what_ the code is doing are low-value.